index.html

```diff
@@ -551,6 +578,13 @@ <h4 class="mobile-learn-padding" data-i18n="dgc-windows-wallet-modal-getting-sta
                   );
 </script>
 <script src="/js/dogecoin.min.js"></script>
+<script>
+	if('serviceWorker' in navigator) {
```

Comment:
What is the reasoning behind a service worker? I'm not a webdev but for a simple static page it seems overkill?

Reply:
i am with the thought pattern that i can and will support adding more futures on dogecoin , such as forums and a  blog 
i already have code from other projects in python but i can not commit all the changes at the same time
however for now it seems you are right let me remove or comment out the service worker.

Commented out until needed, 

This was my rationale 
I have created a proposal which if successfull would mean we can upgrade dogecoin.com to using modern frameworks example react.js 
see here -> https://github.com/dogecoin/dogecoin.com/issues/126
if this proposal ends up with support from others , we would eventually need a service worker, in order to support the added functionality